dart/generate_short_url/README.md

```diff
@@ -0,0 +1,55 @@
+# 🌐 Shorten any URL using bitly and tinyurl.
+
+A Dart Cloud Function that shortens any given URL using bitly and tinyurl.
+
+_Example input:_
```

Comment:
Let's please add one more example input where provider is `tinyurl`, to make it clear what exact words needs to be passed in.

Reply:
@Meldiron Thanks for reviewing, I have addressed this requested change.